src/core_read.cpp

```diff
@@ -177,33 +177,6 @@ bool DecodeHexBlk(CBlock& block, const std::string& strHexBlk)
     return true;
 }
 
-bool DecodeBase64PSBT(PartiallySignedTransaction& psbt, const std::string& base64_tx, std::string& error)
```

Comment:
I think this means you can remove the `#include <psbt.h>` from this file.

Reply:
yep, done